layouts/_default/baseof.html

```diff
@@ -2,6 +2,7 @@
 <html lang="en">
 	<head>
 		<meta charset="utf-8">
+		<meta name="theme-color" content="#013153">
```

Comment:
Can you explain more about this or describe the purpose, or provide a link for me to read? I am unfamiliar with the meta property you added.

Reply:
Kindly Have a look at this page.


https://developers.google.com/web/updates/2014/11/Support-for-theme-color-in-Chrome-39-for-Android

Reply:
Thanks! This is a good idea. But I'd like to make it configurable with a property in the site's config.yaml (or toml). If that item exists, this meta tag would be added, otherwise omitted. Please also take a look at the exampleSite and see where would be the best place to document this. Let me know if I can help!